<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>580</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QListWidget" name="chatList">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>206</y>
     <width>180</width>
     <height>351</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>60</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>48</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="printButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>60</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>print</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>48</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="onduButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>60</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>48</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="redoButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>60</width>
     <height>48</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>48</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>48</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="chatLine">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>559</y>
     <width>148</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="chatPushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>559</y>
     <width>32</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="sLine">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>10</width>
     <height>511</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="onlineList">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>172</y>
     <width>180</width>
     <height>32</height>
    </rect>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
   <property name="gridSize">
    <size>
     <width>20</width>
     <height>32</height>
    </size>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="mainTextEdit">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>70</y>
     <width>606</width>
     <height>509</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="fileInfoLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>70</y>
     <width>180</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>10</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string>fileInfo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="statusInfo">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>560</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="syntaxCombo">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>23</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="colorCombo">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>47</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="highlightLabel">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>21</y>
     <width>52</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>syntaxhighlighter
settings</string>
   </property>
  </widget>
  <widget class="QListWidget" name="versionList">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>70</y>
     <width>200</width>
     <height>508</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="fileInfo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>88</y>
     <width>180</width>
     <height>82</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="windowsIcon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>data/images/title.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="windowsTitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>720</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CoopES edit system</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="minBtn">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>data/images/mini.png</normaloff>data/images/mini.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::DelayedPopup</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>data/images/close.png</normaloff>data/images/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="titleLine">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>800</width>
     <height>2</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="infoLine">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>69</y>
     <width>180</width>
     <height>2</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="shVersions">
   <property name="geometry">
    <rect>
     <x>791</x>
     <y>70</y>
     <width>9</width>
     <height>510</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string>show/hide Versions</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>5</width>
     <height>510</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
   <property name="arrowType">
    <enum>Qt::LeftArrow</enum>
   </property>
  </widget>
  <zorder>chatList</zorder>
  <zorder>saveButton</zorder>
  <zorder>printButton</zorder>
  <zorder>onduButton</zorder>
  <zorder>redoButton</zorder>
  <zorder>chatLine</zorder>
  <zorder>chatPushButton</zorder>
  <zorder>sLine</zorder>
  <zorder>onlineList</zorder>
  <zorder>mainTextEdit</zorder>
  <zorder>fileInfoLabel</zorder>
  <zorder>statusInfo</zorder>
  <zorder>syntaxCombo</zorder>
  <zorder>colorCombo</zorder>
  <zorder>highlightLabel</zorder>
  <zorder>fileInfo</zorder>
  <zorder>windowsIcon</zorder>
  <zorder>windowsTitle</zorder>
  <zorder>minBtn</zorder>
  <zorder>closeBtn</zorder>
  <zorder>titleLine</zorder>
  <zorder>infoLine</zorder>
  <zorder>versionList</zorder>
  <zorder>shVersions</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>onlineList</tabstop>
  <tabstop>chatList</tabstop>
  <tabstop>chatLine</tabstop>
  <tabstop>chatPushButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>printButton</tabstop>
  <tabstop>onduButton</tabstop>
  <tabstop>redoButton</tabstop>
  <tabstop>syntaxCombo</tabstop>
  <tabstop>colorCombo</tabstop>
  <tabstop>mainTextEdit</tabstop>
  <tabstop>versionList</tabstop>
  <tabstop>minBtn</tabstop>
  <tabstop>closeBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>minBtn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>763</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>786</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shVersions</sender>
   <signal>clicked(bool)</signal>
   <receiver>versionList</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>796</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
